terraform/infrastructure/load_balancer.tf

```diff
@@ -82,3 +71,86 @@ resource "aws_lb_target_group" "https" {
     protocol = "TCP"
   }
 }
+
+resource "aws_lb" "app_load_balancer" {
+  name               = "webcms-${var.environment}-alb"
+  internal           = true
+  load_balancer_type = "application"
+
+  subnets         = local.public_subnets
+  security_groups = [data.aws_ssm_parameter.alb_security_group]
+
+  access_logs {
+    bucket  = aws_s3_bucket.elb_logs.bucket
```

Comment:
Do we want to switch the logging over to the bucket provided by Task 7 with this commit?  The bucket is prefixed by the account number, so may be a little more involved.

Reply:
That's a good idea. We don't have a similar bucket in our own AWS environment, so I think we'll probably want to continue creating the current S3 logging bucket for now.

However, as a transitional item, I'm going to create a new variable for the logging bucket and let it default to `null` – by default, it'll use the managed bucket, but you can override it to use the preconfigured bucket instead. We can probably move to fully unmanaged buckets in the same pass when we decommission the Traefik resources.

Reply:
Do we want to wait on that variable before merging this?  If we're working on a deadline to get ALB's up then I'm comfortable merging and getting the variable change later

Reply:
I just pushed the code. Sorry about the delay!